bugsnag-android-core/src/test/java/com/bugsnag/android/JsonUtils.kt

```diff
@@ -21,7 +21,8 @@ internal fun verifyJsonMatches(map: Map<String, Any>, resourceName: String) {
 
 private fun validateJson(resourceName: String, json: String) {
     val whitespace = "\\s".toRegex()
-    val expectedJson = JsonParser().read(resourceName).replace(whitespace, "")
+    val rawJson = JsonParser().read(resourceName)
```

Comment:
What's the reason for this?  Does it allow more access somewhere down the line?

Reply:
This was only done for readability to avoid chaining the `replace()` method call - if it's not clear it can be changed back.